.vscode/settings.json

```diff
@@ -1 +1,3 @@
-{}
+{
+   "typescript.tsdk": "node_modules/typescript/lib"
+}
```

Comment:
This tells vscode to use the typescript (for tsserver) version from the repo instead of the one that's bundled in vscode. 

See: becd0b5e8856381703f48225e92e0992660d8040